config/config.go

```diff
@@ -24,22 +24,23 @@ import (
 
 const (
```

Comment:
General note: the mode for all of the files changed from 644 → 755.

Reply:
i don't know how this happend,what action should i take ?

Reply:
You probably need to use chmod to change the permissions back to 644. Not a big thing, but the less permissions the better. It's also less noise in the PR (less files show as changed).

Reply:
updated

Reply:
Did you push the changes?

Reply:
pushed the changes

Reply:
@rustiever I still see all files with the same code 755.